packages/venia-ui/lib/components/ProductImageCarousel/thumbnail.js

```diff
@@ -66,13 +66,14 @@ const Thumbnail = props => {
     }, [file, isDesktop, label, classes.image]);
 
     return (
-        <button
+        <span
             className={isActive ? classes.rootSelected : classes.root}
             onClick={handleClick}
             type="button"
+            aria-hidden="true"
```

Comment:
Why aria-hidden?
Can't we just make this an AriaButton with an aria label like the others?

Reply:
From my understanding, we are using `aria-hidden="true"` to prevent any accessibility interactions.
So it wouldn't be useful to use AriaButton here.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute

Also, it was transformed to a `span` since we shouldn't be using `aria-hidden` on focusable elements.

However, it shouldn't be `type="button"` but `role="button"` in this case. I will do the update.

Reply:
Also, it might be useful to move the AriaButton component to the root so we can use it elsewhere.
Or maybe use it for every Button?